api/v1/model_handlers.go

```diff
@@ -474,6 +490,76 @@ func (a *Api) podMetrics(request *restful.Request, response *restful.Response) {
 	response.WriteEntity(exportTimeseries(timeseries, new_stamp))
 }
 
+type AggregatedMetricPoints []AggregatedMetricPoint
+
+func (s AggregatedMetricPoints) Len() int {
+	return len(s)
+}
+func (s AggregatedMetricPoints) Swap(i, j int) {
+	s[i], s[j] = s[j], s[i]
+}
+func (s AggregatedMetricPoints) Less(i, j int) bool {
+	// we want the descending order
+	return s[j].Timestamp.Before(s[i].Timestamp)
+}
+
+// podMetrics returns a metric timeseries for a metric of the Pod entity.
+func (a *Api) podMetricsSum(request *restful.Request, response *restful.Response) {
```

Comment:
For this case I would recommend creating a new struct in model/types.go that encapsulates a TimePoint and the count. 

For further reference, on the next release of Heapster the model will always return TimePoints at every resolution for all entities that are active in the cluster, so at that point the sample count might not be as meaningful.

In general, feel free to modify any external type you may need at api/v1/model_types.go .
However, please do not edit the TimePoint type or any existing types in model/types.go, as we are trying to minimize the total memory footprint of the model at this point.